pkg/suites/example.go

```diff
@@ -16,6 +16,10 @@ import (
 func Example(t *testing.T, driver Driver) {
 	actors := make(map[state.Address]state.Actor)
 
+	// TODO this is poor UX, these will maybe need to go somehwere else?
+	gasPrice := big.NewInt(1)
+	gasLimit := big.NewInt(1000)
```

Comment:
Two options (for a follow-up):
- Add state to the producer setting a gas price, and default gas limits in each of the sugar methods
- A functional options pattern to allow optional customization of the sugar methods behaviour